src/igvfd/upgrade/sample.py

```diff
@@ -513,3 +513,13 @@ def in_vitro_system_23_24(value, system):
             f' Please make sure it is correct before removing the notes.'
         )
         value['notes'] = notes.strip()
+
+
+@upgrade_step('primary_cell', '21', '22')
+@upgrade_step('in_vitro_system', '24', '25')
+def sample_21_22(value, system):
+    # https://igvf.atlassian.net/browse/IGVF-2310
+    value['biosample_qualifiers'] = ['6 days calcified']
```

Comment:
I think there should be a check before this so only
```
if 'calcified' in value['biosample_qualifiers']:
value['biosample_qualifiers'].remove('calcified')
value['biosample_qualifiers'].append('6 days calcified')
```

Otherwise this will add 6 days calcified onto every sample